core/src/main/resources/jenkins/split-plugins.txt

```diff
@@ -29,5 +29,8 @@ jdk-tool 2.112 1.0
 # JENKINS-55681
 jaxb 2.163 2.3.0 11
 
+# JENKINS-64107
+sshd 2.281 3.0.1
```

Comment:
I still do not see this on the UC?

Reply:
?

```
➜  wars cat update-center.actual.json | jq '.plugins["sshd"]'
{
  "buildDate": "Feb 23, 2021",
  "dependencies": [],
  "developers": [
    {
      "developerId": "timja"
    }
  ],
  "excerpt": "Adds SSH server functionality to Jenkins, exposing CLI commands through it",
  "gav": "org.jenkins-ci.modules:sshd:3.0.1",
  "labels": [],
  "minimumJavaVersion": "1.8",
  "name": "sshd",
  "popularity": 0,
  "previousTimestamp": "2021-02-22T19:33:24.00Z",
  "previousVersion": "3.0",
  "releaseTimestamp": "2021-02-23T15:53:28.00Z",
  "requiredCore": "2.282-rc30905.2ddf35821c11",
  "scm": "https://github.com/jenkinsci/sshd-plugin",
  "sha1": "xX2ALuJ47hl9sGVwrc6WptJTj+g=",
  "sha256": "Jmpl9bTjReS3qrwwTf3d0CTR8RyB/bZS3u+IQVzvEJg=",
  "title": "SSH server",
  "url": "https://updates.jenkins.io/download/plugins/sshd/3.0.1/sshd.hpi",
  "version": "3.0.1",
  "wiki": "https://plugins.jenkins.io/sshd"
}
```

Reply:
```console
$ curl -sL 'https://updates.jenkins-ci.org/update-center.json?version=2.282' | head -n-1 | tail -n+2 | jq -r '.plugins | .["sshd"] | .version'
3.0
```

Reply:
seems not portable command?

```
head: illegal line count -- -1
```

Reply:
hmm weird, maybe update center is ignoring the version in the dynamic tiers, 'latest' has it

Reply:
Maybe does not refresh until after 2.282 actually comes out. If so, fine.